.dockerignore

```diff
@@ -0,0 +1,14 @@
+#Ignore the following directories
+backend/build/
+backend/.gradle/
+frontend/build/
+frontend/node_modules/
+nginx/
+ops/services/app_functions/report_stream_batched_publisher/functions/node_modules/
```

Comment:
I know we've already talked about this line but could we change this to `ops/**/.terraform/`? I tried building these images locally with your changes and because of the several versions of providers and modules I have installed locally, my machine put *7.3 GB* into memory for the docker context. With the above, that went down to _110 MB_.